src/sentinel.c

```diff
@@ -549,6 +547,26 @@ void initSentinel(void) {
     server.sentinel_config = NULL;
 }
 
+void sentinelReleaseScriptJob(sentinelScriptJob *sj);
+
+void freeSentinel(void) {
```

Comment:
Address sanitizer complains about leaks in sentinel. Added this function to call in prepareForShutdown().